Gemfile.lock

```diff
@@ -0,0 +1,43 @@
+GIT
```

Comment:
You shouldn't commit a Gemfile.lock for a gem

Reply:
TIL - do you have something I can read? Should I be git ignoring it? 🤔

Reply:
Okay - I finally managed to find some clarity: 
- https://github.com/rubygems/rubygems/commit/335a6490503d8e4aa78ab747aa631bbb793a69d4
- [Corresponding blog post](https://bundler.io/blog/2018/01/17/making-gem-development-a-little-better.html)

So the reason a `Gemfile.lock` initially was excluded is so that gems are tested against the being run on a "clean" install in CI, at the cost of developer experience when getting setup with the project. I think optimizing for developer experience is preferred over an extra line in our CI config